docs/pypi-dependencies.md

```diff
@@ -23,12 +55,11 @@ pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 pip.parse(
     hub_name = "my_deps",
     python_version = "3.11",
-    requirements_lock = "//:requirements_lock_3_11.txt",
+    requirements_lock = "//:requirements_lock.txt",
```

Comment:
Please revert. the 3_11 part of the file name is to help emphasize the resolution is 3.11 specific